### PR TITLE
Wiki Suggestions

### PR DESCRIPTION
**Pull request by SidharthK2**
# Adds wiki suggestions feature

- Replaces `Create Wiki` button with `Suggest Wiki` if no  is user logged in.
- Send discord alerts on suggestions

image

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/2255

### CODE REVIEW

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
- The heading text is too close to the input inbox.
- Let's make the description text 2 lines and the dialogue box a little wider. 
The size in the design is not too big (note that it's not a widget, so it doesn't have to have such a small size).
image

**Review by Simplypheyie96 — APPROVED** [review]

File: src/components/Layout/Navbar/SuggestWiki.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        <Box mt={-2}>
+          <Text fontSize="sm">
+            Our wiki database is growing to cater to diverse interests in the
+            crypto-space. Have a topic or area that you'd like us to cover? Feel
+            free to suggest wikis!
+          </Text>
+          <Textarea
+            value={input}
+            onChange={handleInputChange}
+            placeholder="details"
```
Comment (Adebesin-Cell) [inline src/components/Layout/Navbar/SuggestWiki.tsx:89]: ``Details```

File: src/components/Layout/Navbar/DesktopNav.tsx [hunk 2]
```diff
@@ -19,12 +19,14 @@ import { ChevronDownIcon } from '@chakra-ui/icons'
...
 import { RootState } from '@/store/store'
 import useLanguageChange from '@/hooks/useLanguageChange'
+import { getUserAddressFromCache } from '@/utils/WalletUtils/getUserAddressFromCache'
 
 const DesktopNav = () => {
   const router = useRouter()
   const [visibleMenu, setVisibleMenu] = useState<number | null>(null)
   const lang = useSelector((state: RootState) => state.app.language)
   const { handleLangChange } = useLanguageChange()
+  const userAddress = getUserAddressFromCache()
```
Comment (Adebesin-Cell) [inline src/components/Layout/Navbar/DesktopNav.tsx:29]: It kind of glithces, try to use useAccount to get address

**Comment by Simplypheyie96** [comment]
@SidharthK2 I missed this.
let's make sub text 2 lines just like the design. The height of the modal too shouldn't be so short. 
The height to widgth ratio is weird.
image
--- 
The design
image

**Review by Softdev1 — APPROVED** [review]
This looks alot good

**Pull request merged**